org.kde.kbibtex.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "org.kde.kbibtex",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbibtex",
+    "rename-icon": "kbibtex",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=cups",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "poppler",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+                "-DCMAKE_INSTALL_INCLUDEDIR=/app/include",
+                "-DENABLE_LIBOPENJPEG=none",
+                "-DENABLE_BOOST=OFF",
```

Comment:
make sure you don't need the splash backend, because disabling  boost makes it terribly slow.

Reply:
If Boost was disabled due to the added build time length, then maybe try building only the container library and without debug symbols.

```
{
    "name": "boost",
    "buildsystem": "simple",
    "build-commands": [
        "./bootstrap.sh",
        "./b2 --with-container variant=release link=shared debug-symbols=off runtime-link=shared --prefix=${FLATPAK_DEST} install"
    ],
    "sources": [
        {
            "type": "archive",
            "url": "https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.bz2",
            "sha256": "475d589d51a7f8b3ba2ba4eda022b170e562ca3b760ee922c146b6c65856ef39",
            "x-checker-data": {
                "type": "anitya",
                "project-id": 6845,
                "stable-only": true,
                "url-template": "https://boostorg.jfrog.io/artifactory/main/release/$major.$minor.$patch/source/boost_${major}_${minor}_$patch.tar.bz2"
            }
        }
    ]
}
```

Reply:
you only need to copy the headers. and you don't even need to update it. It needs a very specific module that is header only.

or you can just ignore and if you use the splash backend, suffer swoliness. After all Ubuntu still build it like that.

Reply:
I went to check the source to me it looks like it doesn't use the Splash backend.

Reply:
See example in https://github.com/flathub/org.kde.okular/blob/master/org.kde.okular.json

Reply:
I added boost and openjpeg just to be sure nothing is missing. Disabled poppler tests to save some build time.